### ISSUE TITLE
[Bug] Failing to evaluate model - unexplained error 

### ISSUE THREAD

**Issue by ofersabo** [codalab/codalab-competitions#2739]
Hi,

I am using CodaLab CLI version 0.5.4

Recently, I have been working to submit my trained model's prediction on a  competition test set.   
I created my own docker to mimic my local environment.

The program's first step is downloading a pre-trained model from a server. To this end,  I ask a codalab server with a network connection. This enables the process to successfully download the model, but just after that, the process fails. 

The stderr file doesn't hold any information regarding the failure reason.   
I don't see any way to debug this failure, I note that in my local machine the program runs smoothly.   

attached is a screenshot from the worksheet sidebar, which states that the process is killed.

![error](https://user-images.githubusercontent.com/31179968/72821022-9282a900-3c78-11ea-8860-ba8ec9f67227.png)

**Comment by Didayolo** [comment]
Hi,

The Codalab CLI tool is related to [Codalab Worksheets](https://github.com/codalab/codalab-worksheets). 

It is a branch of the project separated from Codalab competitions (here). 
[Codalab competitions](https://competitions.codalab.org) is a platform hosting challenges while [Codalab worksheets](https://worksheets.codalab.org/) aims at running ML experiments on the cloud.

You can raise an issue [here](https://github.com/codalab/codalab-worksheets/issues).
